packages/core/contracts/insured-bridge/BridgeDepositBox.sol

```diff
@@ -204,7 +192,7 @@ abstract contract BridgeDepositBox is Testable, Lockable {
         // Else, it is a normal ERC20. In this case pull the token from the users wallet as per normal.
         // Note: this includes the case where the L2 user has WETH (already wrapped ETH) and wants to bridge them. In
         // this case the msg.value will be set to 0, indicating a "normal" ERC20 bridging action.
-        else TokenHelper.safeTransferFrom(l2Token, msg.sender, address(this), amount);
+        else IERC20(l2Token).safeTransferFrom(msg.sender, address(this), amount);
```

Comment:
somewhat unfortunate artifact of the transfer helper removal is the need to cast these addresses to interfaces.